modules/statics/src/base.ts

```diff
@@ -33,7 +33,7 @@ export enum CoinFamily {
   DOT = 'dot',
   ETH = 'eth',
   ETH2 = 'eth2',
-  ETHW = 'ethw',
+  ETHW = 'eth',
```

Comment:
if `ETHW` is indeed meant to just be part of the Ethereum CoinFamily, then this enum should be deleted entirely & the account ctor should be updated accordingly.

Reply:
We should keep this as `ethw`. It is an eth fork like `etc`, then they can evolve separately so we consider them different families.
@benpreiser , as you pointed we have a mismatch in `getFamily` implementation of ethw. It should return `ethw`.